client/src/lib/helper.js

```diff
@@ -1,6 +1,6 @@
 import $ from 'jquery';
 
-import BASE_URL from './url';
+const BASE_URL = process.env.BASE_URL ? process.env.BASE_URL : '';
```

Comment:
im assuming the base url would be the http:// part of the url? what will happen if its     ' '      that becomes BASE_URL? will it just be /restaurants/id/reservations/date?

Reply:
yup!  If it goes to the blank string default then all of the AJAX URLs will become relative URLs